archunit/src/main/java/com/tngtech/archunit/core/domain/JavaMethod.java

```diff
@@ -125,6 +125,14 @@ public String getDescription() {
         return "Method <" + getFullName() + ">";
     }
 
+    @PublicAPI(usage = ACCESS)
+    public boolean isOverridden() {
+        return getOwner().getAllRawSuperclasses().stream()
```

Comment:
If we want to conisder overridden methods from interfaces as well, we can replace this stream with
```java
Streams.concat(getOwner().getAllRawSuperclasses().stream(), getOwner().getAllRawInterfaces().stream())
```